cypress/e2e/custom-view-template-starter/channels.cy.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  ENTRY_POINT_CUSTOM_VIEW_TEMPLATE_STARTER,
+  URL_CUSTOM_VIEW_TEMPLATE_STARTER,
+} from '../../support/urls';
+
+describe('Channels', () => {
+  beforeEach(() => {
+    cy.loginToMerchantCenter({
+      entryPointUriPath: ENTRY_POINT_CUSTOM_VIEW_TEMPLATE_STARTER,
+      initialRoute: URL_CUSTOM_VIEW_TEMPLATE_STARTER,
+    });
```

Comment:
Nit: when developing Custom Views, some of these options are not really relevant for users.
Instead of explicitly passing these options we could maybe have them as default within the command.

For that we might want to add a new command e.g. `loginToMerchantCenterForCustomView` or something. Then the `entryPointUriPath` does not need to be requested and the "base path" can be predefined.

Something to think about for a follow up.

Reply:
Sure, great point, I'll work on that.

Reply:
Thanks, you can look into that for a follow up. No need to do it in this PR